api/howToSupport.js

```diff
@@ -42,14 +41,17 @@ const unwrapVolunteerOpening = volunteerOpening => ({
 })
 
 export async function fetchBecomeAVolunteerPage(locale) {
-  const content = await fetchMemoizedSingleEntry('pageAktivWerden', locale) || {}
+  const content = (await fetchMemoizedSingleEntry('pageAktivWerden', locale)) || {}
   return {
     ...content,
-    regionalGroups: unwrapRegionalGroups(content && content.regionalGroups),
     section1TeamMember: unwrapTeamMember(content && content.section1TeamMember),
     volunteerOpenings: content && content.volunteerOpenings.map(unwrapVolunteerOpening),
     bannerButtonUrl: unwrapPageUrl(content.bannerButtonUrl),
     section1ReferenceList: unwrapTestimonials(content.section1ReferenceList),
+    section4VolunteerOpenings:
+      content && content.section4VolunteerOpenings.map(unwrapVolunteerOpening),
+    section5ReferenceList: unwrapTestimonials(content.section5ReferenceList),
```

Comment:
I think we need a `content &&` here or a check inside `unwrapTestimonials()`. Both here and 3 lines up.

Reply:
Ohh yeah you are right. I just forgot to put it to my commit 😊